src/style.css

```diff
@@ -2,2 +1,82 @@
+body {   font-family: 'Roboto', sans-serif;
   margin: 0;
+  background-color: aliceblue;
+  font-size: 12px;
+  text-transform: uppercase;
 }
+.header{
+  background-color: #fff;
+width:  100%;
+height: 60px ;
+display: flex;
+justify-content: space-between;
+
+}
+
+.image{
+  height: 40px;
+  width: 40px;
+  background-color: #fff;
+margin-left: 50px;
+}
+
+.nav{
+  display: inline;
+  text-decoration: none;
+  white-space: nowrap;
+  margin-right: 50px;
+
+
+}
+
+
+.is-active{
+  color:#00ACDC ;
+  position: relative;
+
+  }
+
+  .laptops{
+    color: #00ACDC
```

Comment:
There is a missing semicolon at the end of the `color` property in the `.laptops` class. This might cause issues in some browsers. Ensure all CSS properties are properly terminated with a semicolon.